apps/calendar/js/views/month.js

```diff
@@ -105,12 +108,25 @@ Month.prototype = {
         this.controller.selectedDay = date;
         break;
       case 'dbltap':
-        this.app.go('/day/');
+        // make sure we discard double taps that started on a different day
+        if (this._lastTarget === target) {
+          this._goToAddEvent();
+        }
         break;
       case 'monthChange':
         this.changeDate(e.data[0]);
         break;
     }
+    this._lastTarget = target;
+  },
+
+  _goToAddEvent: function(date) {
+    // slight delay to avoid tapping the elements inside the add event screen
```

Comment:
we would still need to block interaction during the transition. I would say we keep it like this for now and implement transitions later as a separate bug.